api4/user_test.go

```diff
@@ -5502,11 +5502,13 @@ func TestGetThreadsForUser(t *testing.T) {
 
 		var rootIds []*model.Post
 		for i := 0; i < 30; i++ {
+			// nolint:staticcheck
 			time.Sleep(1)
```

Comment:
Can we please remove all of these `time.Sleep(1)` calls? I think all of these are redundant. If it surfaces any underlying issues in the test, we would want to fix that rather than using these sleeps.

Reply:
I remove all of these time.Sleep(1), the test was passed in local.